src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -213,6 +204,37 @@ function createEmscriptenModuleInstance(loadAssemblyUrls: string[], onReady: ()
   });
 
   return module;
+
+  async function addResourceAsAssembly(dependency: LoadingResource) {
+    const runDependencyId = `blazor:${dependency.name}`;
+    Module.addRunDependency(runDependencyId);
+
+    try {
+      // Wait for the data to be loaded and verified
+      const dataBuffer = await dependency.data;
+
+      // Load it into the Mono runtime
+      const data = new Uint8Array(dataBuffer);
+      const heapAddress = Module._malloc(data.length);
+      const heapMemory = new Uint8Array(Module.HEAPU8.buffer, heapAddress, data.length);
+      heapMemory.set(data);
+      mono_wasm_add_assembly(dependency.name, heapAddress, data.length);
+      MONO.loaded_files.push(toAbsoluteUrl(dependency.url));
+    } catch (errorInfo) {
+      // If it's a 404 on a .pdb, we don't want to block the app from starting up.
+      // We'll just skip that file and continue (though the 404 is logged in the console).
+      // This happens if you build a Debug build but then run in Production environment.
+      const isPdb404 = errorInfo instanceof XMLHttpRequest
```

Comment:
Just realised this is now wrong since we’re no longer using XMLHttpRequest. 

I think we could just let it fail in this scenario anyway now.

Reply:
Fixed